TrkHitReco/fcl/prolog.fcl

```diff
@@ -9,33 +9,51 @@ PBTFSD : {
 
 makeSH : {
   module_type             : StrawHitReco
-  minimumTime             : 0.0 # accept all digis
-  maximumTime             : 100.0e3 # allow offspill events
+  MinimumTime             : 420.0 # ns configuration for OnSpill
+  MaximumTime             : 1690.0 # ns
+  MinimumEnergy           : 0.0001 # MeV
+  MaximumEnergy           : 0.005 # MeV
+  MinimumRadius           : 0.0 # mm
+  MaximumRadius           : 1000.0 # mm
   FitType                 : 1
   FilterHits              : false
   WriteStrawHitCollection : true
-  ProtonBunchTimeTag  : "EWMProducer"
+  ProtonBunchTimeTag      : "EWMProducer"
   StrawDigiCollectionTag  : "makeSD"
   StrawDigiADCWaveformCollectionTag : "makeSD"
-  CaloClusterCollectionTag  : "CaloClusterFast"
+  UseCalorimeter          : false
+  clusterDt               : 100.0 # ns
+  CaloClusterCollectionTag: "notUsed"
+  FlagCrossTalk           : false
+  crossTalkEnergy         : 0.007 # MeV
+  crossTalkMinimumTime    : -1 # ns
+  crossTalkMaximumTime    : 100 # ns
 }
 
 # combine hits in a panel
 makePH : {
   module_type           : CombineStrawHits
   TestFlag              : true
-  TestRadius            : true
+  FilterHits            : false
   ComboHitCollection    : makeSH
-  StrawHitSelectionBits : ["EnergySelection","TimeSelection"]
-  StrawHitMask          : []
-  MaxDt                 : 40
-  UseTOT                : false
+  StrawHitSelectionBits : ["EnergySelection","TimeSelection","RadiusSelection"]
```

Comment:
This was probably discussed during some meetings that I didn't attend, so I apologize for addressing questions probably already been discussed. 
- wouldn't be more accurate to define a parameter that masks the N-straws closer to the solenoid axis? I'm not sure that using the radial distance as a proxy provides the same results.
- I remember that the "radiusSelection" was motivated by the poor resolution that is expected for the straws in that region. Is that still true even if we use the ToT corrected time?